django_project/ta_system/admin.py

```diff
@@ -26,35 +26,42 @@
 
 class CustomAdminSite(AdminSite):
 
-    site_title  = "Boston College TA Application System"
+    site_title = "Boston College TA Application System"
     site_header = "Boston College TA Application System"
     index_title = "System Admin"
 
     def get_urls(self):
         urls = super().get_urls()
         urls = [
             path('', self.admin_view(self.index)),
-            path('course_data_upload', 
-                self.admin_view(self.course_data_upload), 
-                name='course_data_upload'),
-            path('applicant_data_upload', 
-                self.admin_view(self.applicant_data_upload), 
-                name='student_data_upload'),
+            path('course_data_upload',
+                 self.admin_view(self.course_data_upload),
+                 name='course_data_upload'),
+            path('applicant_data_upload',
+                 self.admin_view(self.applicant_data_upload),
+                 name='student_data_upload'),
             path('assignment_data_download',
-                self.admin_view(self.assignment_data_download),
-                name='assignment_data_download')
+                 self.admin_view(self.assignment_data_download),
+                 name='assignment_data_download'),
+            path('change_system_status',
+                 self.admin_view(self.change_system_status),
+                 name='change_system_status')
         ] + urls
         return urls
 
     def index(self, request):
         app_list = self.get_app_list(request)
+        status_list = models.SystemStatus.objects.order_by('id')
+        status_list = status_list.reverse()
         context = {
             **self.each_context(request),
             'title': self.index_title,
             'app_list': app_list,
             'course_data_upload_form': forms.CourseDataUploadForm(),
             'applicant_data_upload_form': forms.ApplicantDataUploadForm(),
-            'assignment_data_download_form': forms.AssignmentDataDownloadForm()
+            'assignment_data_download_form': forms.AssignmentDataDownloadForm(),
+            'system_status': status_list,
```

Comment:
Didn't you say you had removed the system_status, since it isn't actually being used in admin/index.html?

Reply:
code deleted